src/dolphin/atmosphere/ionosphere.py

```diff
@@ -117,11 +117,11 @@ def estimate_ionospheric_delay(
             )
             continue
 
-        reference_date = (ref_date,)
-        secondary_date = (sec_date,)
+        reference_date = next(key for key in slc_files if ref_date in key)
```

Comment:
maybe a comment as to how this is different than the above? Being out of the loop, I can't quite tell how the SLC date would be different than the two interferogram dates

Reply:
Sure, I added a comment